platform/mbed_error.c

```diff
@@ -74,14 +74,19 @@ static unsigned int compute_crc32(const void *data, int datalen)
     unsigned int crc = 0; /* CRC value is 32bit */
     unsigned char *buf = (unsigned char *)data;//use a temp variable to make code readable and to avoid typecasting issues.
 
-    for (; datalen > 0; datalen--) {
+    for (; datalen > 0; datalen--)
+    {
```

Comment:
Coding style requires 1 TBS for  if, else, while, for statements, see  https://os.mbed.com/teams/SDK-Development/wiki/mbed-sdk-coding-style
Please revert this change

Reply:
> astyle --style=allman applied to pass astyle test

Use astylerc file in the root, defines all rules needed - just run it on your changes

Reply:
@0xc0170 
I ran atstyle based on the rc file and this is what I got.
```
andcho01@bc-a1-1-3:~/code/mbed-os$ astyle -n --options=.astylerc platform/mbed_error.c
Invalid option file options:
max-continuation-indent=120

For help on options type 'astyle -h'

Artistic Style has terminated
```

Reply:
I believe I  saw this error earlier - what version of astyle you use? Update if older, they do not provide this option